service/src/main/java/uk/gov/hmcts/reform/fpl/model/MigratedChildren.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class MigratedChildren {
+    private final ChildParty party;
+
+    @JsonCreator
```

Comment:
No need to have `@JsonCreator` and `@JsonProperty`. Please remove. If any problems encountered make sure you use `ObjectMapper` instantiated via framework. Do not instantiate it yourself.

Reply:
May need a hand with this @geek-soft